src/GitHub.Exports/Services/VSServices.cs

```diff
@@ -124,9 +124,10 @@ static IEnumerable<ISimpleRepositoryModel> PokeTheRegistryForRepositoryList()
                             if (path != null)
                                 return new SimpleRepositoryModel(path);
                         }
-                        catch (Exception ex)
+                        // no sense spamming the log, the registry might have ton of stale things we don't care about
+                        catch (Exception)
                         {
-                            VsOutputLogger.WriteLine(string.Format(CultureInfo.CurrentCulture, "Error loading the repository from the registry '{0}'", ex));
+                            //VsOutputLogger.WriteLine(string.Format(CultureInfo.CurrentCulture, "Error loading the repository from the registry '{0}'", ex));
```

Comment:
If we don't think we need to log here, let's just remove the commented out code and replace it with the comment.